content/log/processing.md

```diff
@@ -0,0 +1,199 @@
+
+Check to wich subset of logs your pipeline is applied to in the log list displayed: 
+
+{{< img src="log/processing/pipeline_filters.png" alt="Pipelines processor sketch" style="width:90%;">}}
+
+### Integrations Pipelines
+
+Integration pipelines are automatically installed when you activate an [integration](https://app.datadoghq.com/account/settings)
+
+Those pipeline are read-only but can be cloned for further customisation:
+
+{{< img src="log/processing/cloning_pipeline.png" alt="Cloning pipeline" >}}
+
+## Processors
+
+**A processor executes a well identified action on a log most often related to parsing & enrichment.**
+
+The different kind of processors are explained below.
+
+### Grok Parser
+
+Create custom grok rules to parse the full message or a specific attribute of your raw event:
+
+{{< img src="log/processing/parser.png" alt="Parser" >}}
+
+Find more about this in our dedicated [parsing section](/log/parsing)
+
+### Log Date Remapper 
+
+By default (as explained [here](/log/#the-date-attribute)) Datadog has a couple of reserved attributes that are, by default, considered as the date reference for the main timeline.
+
+However, if you have your own date attribute that does not belong to this list, define it as the official log timestamp with the log date remapper processor:
+{{< img src="log/processing/log_date_remapper.png" alt="Log date remapper" >}}
+
+Otherwise the default timestamp is the reception time of the log by the Datadog intake API.
+
+### Log Severity Remapper
+
+Use this processor if you want to assign some attributes as the official severity, just enter the attribute path in the processor tile as follow:
+
+{{< img src="log/processing/severity_remapper_processor_tile.png" alt="Severity remapper processor tile" >}}
+
+It transforms this log:
+{{< highlight json >}}
+{
+"message":"john clicked on a link : www.datadoghq.com for app dd_app",
+"hostname":"host_1",
+"level":"info"
+}
+{{< /highlight >}}
+
+Into this log:
+{{< highlight json >}}
+{
+"message":"john clicked on a link : www.datadoghq.com for app dd_app",
+"hostname":"host_1",
+"severity":"info"
+}
+{{< /highlight >}}
+
+However, beware that each incoming severity value is mapped as follows:
+
+* If this is an integer from 0 to 7 we map it to the [syslog severity standards](https://en.wikipedia.org/wiki/Syslog#Severity_level)
+* If it starts with **emerg** or **f** (case unsensitive) we map it to **emerg (0)**
+* If it starts with **a** (case unsensitive) we map it to **alert (1)**
+* If it starts with **c** (case unsensitive) we map it to **critical (2)**
+* If it starts with **err** (case unsensitive) we map it to **error (3)**
+* If it starts with **w** (case unsensitive) we map it to **warning (4)**
+* If it starts with **n** (case unsensitive) we map it to **notice (5)**
+* If it starts with **i** (case unsensitive) we map it to **info (6)**
+* If it starts with **d**, trace or verbose (case unsensitive) we map it to **debug (7)**
```

Comment:
Are 'trace' and 'verbose' to be taken literally? If so, why not bold them like the other literals?

Reply:
👍  good catch